### DISCUSSION TITLE
returned value- rest API

### DISCUSSION THREAD

**Discussion by ATHENA12D** [Budibase/budibase#12743] [Help] [answered]
Hi, I need to calculate the total number of word "Test" from http://my-example/text.html using the rest api, I mean when i make a GET request on the url http://my-example/text.html , the return value should be only the total numbers of the word "test" "
How can i do it this?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @ATHENA12D,

Is this a Budibase question or are you just asking about your own API?

If its related to Budibase, you would use JS in the transformer to return back the number of times test is contained within the objects inside your array.
